src/c++/pyndarray.h

```diff
@@ -0,0 +1,8 @@
+#if BOOST_VERSION < 106500
+  typedef typename boost::python::numeric::array pyndarray;
+  //namespace np = boost::python::numeric;
+#else
+  #include <boost/python/numpy.hpp>
+  typedef typename boost::python::numpy::ndarray pyndarray;
+  namespace np = boost::python::numpy;
```

Comment:
This namespace seems to be used only in two places, perhaps better to just keep it explicit there?